### ISSUE TITLE
Some extension README code snippets are not displayed

### ISSUE THREAD

**Issue by martinwork** [microsoft/pxt-microbit#5309]
**Describe the bug**

Some extension README code snippets are not displayed.

Some are displayed in a browser tab, but not when opened in MakeCode using the Javascript Explorer. 

**To Reproduce**
1. Edit project https://makecode.microbit.org/_RAA39dEfV1iP
2. Right click on 'left turn'
3. Click 'Help'

**Expected behavior**
Code snippets are visible.

**Screenshots**
![image](https://github.com/microsoft/pxt-microbit/assets/989126/2459220a-0430-41b1-82fd-4a3bdef2a526)

**microturtle in separate browser tab**

![image](https://github.com/microsoft/pxt-microbit/assets/989126/a1ad685e-3e2c-4cbb-8011-ea13290d5d0f)

**microturtle in MakeCode**

![image](https://github.com/microsoft/pxt-microbit/assets/989126/74cc7297-9284-4d34-9ae3-a5a7e02bfa82)

**Sonar in separate browser tab**

![image](https://github.com/microsoft/pxt-microbit/assets/989126/0970deae-3ada-4143-9068-088992b68cb7)

**Sonar in MakeCode**

![image](https://github.com/microsoft/pxt-microbit/assets/989126/2dbf3aee-622c-4760-a85f-6c090966d321)

**Kitronik ServoLite works in browser tab**

![image](https://github.com/microsoft/pxt-microbit/assets/989126/9d415456-eaa9-4e2d-a86b-7847e46b0ba5)

**But not in MakeCode**

![image](https://github.com/microsoft/pxt-microbit/assets/989126/ab26623b-5784-41fb-8fa7-f37b4866e222)

**micro:bit version (please complete the following information):**
not hardware related

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] 114.0.5735.199 (Official Build) (64-bit)

**Comment by srietkerk** [comment]
This is now fixed. This was a bug that I created when I updated pxt-microturtle and pxt-sonar last year to update their thumbnails in the extension gallery. I bumped their `pxt.json` versions but did not create tags for the updates, so when our docs were trying to get the data from the latest version, there were errors and the jres for the blocks wasn't getting generated.

I have now bumped the tags for pxt-microturtle and pxt-sonar, so this is fixed in live already as well.
